WTraining/app/helpers/bcryptjs.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+const { hashSync, genSaltSync } = require('bcryptjs');
+const { SALT } = require('../constants/constants');
+
+const generateHash = password => hashSync(password, genSaltSync(SALT));
+
+module.exports = { generateHash };
```

Comment:
I prefer to use `exports. generateHash` instance of `module.exports = { generateHash };`. I think visually it is more understandable